requirements.txt

```diff
@@ -7,7 +7,9 @@ flask
 flask-openid
 flask-wtf
 markupsafe
-python-openid
+# TODO: Make next 2 lines conditional on python version
+# python-openid
+python3-openid
```

Comment:
Neither, I need to replace these lines with:

    python-openid; python_version < '3.0' 
    python3-openid; python_version >= '3.0' 

That way it will get the correct version regardless.